src/content/en/updates/2019/06/chromium-chronicle-3

```diff
@@ -0,0 +1,64 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Chromium Chronicle, a monthly series geared specifically to Chromium developers — the developers who build the browser.
+
+{# wf_updated_on: 2019-06-18 #}
+{# wf_published_on: 2019-06-25 #}
+{# wf_tags: chromium-chronicle #}
+{# wf_featured_image: /web/updates/images/generic/cr-chron.jpg #}
+{# wf_featured_snippet: Tests are critical because they find bugs and regressions, enforce better designs and make code easier to maintain. This month, we take a look at how to conduct thorough tests with Gerrit #}
+{# wf_blink_components: N/A #}
+
+<style>
+  body:not(.devsite-dark-code) pre.prettyprint.cc-bad {
+    background-color: #fff7f7;
+  }
+  body:not(.devsite-dark-code) pre.prettyprint.cc-good {
+    background-color: #f7fff7;
+  }
+</style>
+
+# The Chromium Chronicle: Code Coverage in Gerrit {: .page-title }
+
+**Episode 3:** June 2019
+
+*by Yuke, Roberto and Sajjad in Mountain View*
+
+Tests are critical because they find bugs and regressions, enforce better
+designs and make code easier to maintain. Code coverage helps you ensure
+your tests are thorough.
+
+Chromium CLs can show a line-by-line breakdown of test coverage. You can
+use the code coverage trybot to ensure you only submit well-tested code.
+
+**Insert Image**
```

Comment:
Not sure how to insert an image. Help here would be much appreciated.

Reply:
To do two side by side images akin what's in the doc

```
<img src="/web/updates/images/2019/06/LEFT_FILENAME.ext" class="attempt-left">
<img src="/web/updates/images/2019/06/RIGHT_FILENAME.ext" class="attempt-right">
<div class="clearfix"></div>
```

Then, drop the image files in `src/content/en/updates/images/2019/06/`, and add them to the PR.

Reply:
Done! Please let me know if I did this correctly.